src/main/java/com/onaio/steps/helper/DatabaseHelper.java

```diff
@@ -60,6 +60,9 @@ public void onUpgrade(SQLiteDatabase db, int oldVersion, int newVersion) {
         }
     }
 
+    @Override
+    public void onDowngrade(SQLiteDatabase db, int oldVersion, int newVersion) { }
```

Comment:
@owais-vd  are you sure we do not need to handle this with an actual query that restores the db to the state that it was on the oldVersion? Which in the case of this [issue](https://github.com/onaio/steps-app/issues/168) was 2 and we are downgrading to 1?

Reply:
yes, we don't need to handle a downgrade because if we drop some columns in the downgrade method that was added in the previous upgrade that means we also refactor the code because some logic is dependent on some columns. However, if need to tweak a database in the future then we can do it in the `onUpgrade` method.